docs/Reference/Validation-and-Serialization.md

```diff
@@ -397,7 +397,7 @@ configuration](https://github.com/fastify/ajv-compiler#ajv-configuration) is:
 {
   coerceTypes: 'array', // change data type of data to match type keyword
   useDefaults: true, // replace missing properties and items with the values from corresponding default keyword
-  removeAdditional: true, // remove additional properties
+  removeAdditional: true, // To remove additional properites refer to: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```

Comment:
```suggestion
  removeAdditional: true, // remove additional properties if additinalProperties is set to false, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```

Reply:
If we're going the detailing path, I'd suggest something like `// remove additional properties if additinalProperties is set to false in JSON Schema, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties` 😁